security/pkg/nodeagent/sds/server.go

```diff
@@ -144,19 +133,10 @@ func (s *Server) initWorkloadSdsService(options *ca2.Options) error { //nolint:
 	return nil
 }
 
-func (s *Server) grpcServerOptions(options *ca2.Options) []grpc.ServerOption {
+func (s *Server) grpcServerOptions() []grpc.ServerOption {
 	grpcOptions := []grpc.ServerOption{
 		grpc.MaxConcurrentStreams(uint32(maxStreams)),
 	}
 
-	if options.CertFile != "" && options.KeyFile != "" {
-		creds, err := credentials.NewServerTLSFromFile(options.CertFile, options.KeyFile)
```

Comment:
Do you know what is the context of adding this logic. What scenario was that?

Reply:
no clue, its from the original SDS pr

Reply:
I see, this is probably safe to remove. Thanks.